src/app/pages/IdxPage/index.jsx

```diff
@@ -50,20 +50,22 @@ const IdxPage = ({
   mostReadEndpointOverride,
   radioScheduleEndpointOverride,
 }) => {
-  const {
-    mostRead: { onIdxPage },
-  } = useContext(ServiceContext);
+  const { mostRead, lang, radioSchedule, service } = useContext(ServiceContext);
+
   const groups = path(['content', 'groups'], pageData);
   const title = path(['metadata', 'title'], pageData);
-  const lang = path(['metadata', 'language'], pageData);
   const summary = path(['metadata', 'summary'], pageData);
   const seoTitle = path(['promo', 'name'], pageData);
-
-  const { radioSchedule } = useContext(ServiceContext);
   const radioScheduleData = path(['radioScheduleData'], pageData);
   const radioScheduleOnIdxPage = path(['onIdxPage'], radioSchedule);
+  const mostReadOnIdxPage = path(['onIdxPage'], mostRead);
   const radioScheduleIdxPosition = path(['idxPagePosition'], radioSchedule);
 
+  // The /ukraine_in_russian page, has a mix of ukrainian and russian text
```

Comment:
```suggestion
  // Some ukrainian pages, including '/ukrainian/ukraine_in_russian', have headers/footers in Ukrainian, but the main content in Russian.
```
It'd be good to generalise this comment, since some of the upcoming STY assets will also have multiple languages in the same way.